other/file_utils.py

```diff
@@ -0,0 +1,129 @@
+import json
+import sys
+
+
+def touch_file(file_name):
```

Comment:
Type hints?  Doctests?  Algorithmic functions should not print().  See CONTRIBUTING.md.

Reply:
Changes done and sent for review